bokeh/command/subcommands/serve.py

```diff
@@ -917,8 +917,14 @@ def show_callback() -> None:
             if server.address is not None and server.address != '':
                 address_string = server.address
 
+            if server_kwargs['ssl_certfile'] and (
+                    server_kwargs['ssl_certfile'].endswith('.pem') or server_kwargs['ssl_keyfile']):
```

Comment:
```suggestion
            if server_kwargs['ssl_certfile'] and (server_kwargs['ssl_certfile'].endswith('.pem') or server_kwargs['ssl_keyfile']):
```
we have a fairly long line length and prefer to keep conditions to a single-line where reasonable